### PR TITLE
Slight redesign of Valhalla Vendors and Chemistry. Adds FC and Synth to Valhalla.

### PR DESCRIPTION
**Pull request by KM-Catman**
## About The Pull Request

Removes several items in Valhalla with broken sprites. (Armor pieces, Lockers, Books, etc)
Replaces the Prep room doors with their Marine prep room versions.
Makes all the walls and windows of the Valhalla testing room unbreakable (Primarily to help Perma EORD Valhalla players).
Adds a Valhalla Field Commander and his respective prep room.
Adds a Valhalla Synth and it's respective prep broomcloset.
Compresses Valhalla Chemistry and adds a new primary Vendor spot.
Adds a fluttering friend to a small room in Valhalla.
Materializes the ▋▋▋▋▋ in the Chaplain's closet.

## Why It's Good For The Game

Field Commander was requested to be added due to it's unique vendor. Synth is added due to some players fearing to play (and subsequently learn) it due to it having one slot. Unbreakable walls and windows is to preserve at least most of valhalla during Perma EORD. Chemistry had much too large a spot, so I shrunk it down and added Pill bottle boxes. Used the space from shrinking Chemistry to put the vendors I removed when making the FC Prep room.

## Changelog

:cl: Meowosers
add: Field Commander and the Synthetic have been added to Valhalla
tweak: Valhalla's Chemistry has been shrunken to make room for Vendors
tweak: The Prep room in Valhalla has had it's walls reinforced
fix: Removed several broken sprites from the Valhalla station
/:cl:

### CODE REVIEW

**Review by RipGrayson — APPROVED** [review]

**Comment by KM-Catman** [comment]
I'd prefer if this wasn't merged yet. Grayson has advised that I combine it with a separate Valhalla change first.

**Comment by KM-Catman** [comment]
Alright, added a shitload to the PR. All done now

**Review by RipGrayson — CHANGES_REQUESTED** [review]
I'm going to exercise my maintainer given privilege of complaining about meaningless cosmetic details.

Please make the floor tiles here red instead of black.
wall10

Something like this anyway, but it doesn't have to be exact.
wall11

**Review by RipGrayson — APPROVED** [review]

**Comment by KM-Catman** [comment]
image
image
image

**Review by RipGrayson — CHANGES_REQUESTED** [review]
wall15
You have space tiles here, they need fixed before I can merge

**Comment by KM-Catman** [comment]
> 

Damnit, on it. Will this add another 24 hour delay or no?

**Comment by KM-Catman** [comment]
There, space sufficiently duct taped

File: code/datums/jobs/job/fallen.dm
Comment (ivanmixo) [inline code/datums/jobs/job/fallen.dm:75]: Yeah?

**Pull request merged**
